app/views/tags/index.html.erb

```diff
@@ -19,25 +19,52 @@
   <meta name="twitter:image:src" content="<%= SiteConfig.main_social_image %>">
 <% end %>
 
-<div class="user-profile-header tag-header bg-transparent">
-  <div class="tag-or-query-header-container">
+<style>
+  .tag-card:not(:hover) {
+    box-shadow: 0 0 0 1px var(--card-border) !important;
+    border: 2px solid transparent !important;
+  }
+
+  .tag-card:not(:hover) .crayons-btn--secondary.follow-action-button {
+    background: var(--button-secondary-bg) !important;
+    color: var(--button-secodnary-color) !important;
+  }
+</style>
+
+<div class="crayons-layout crayons-layout--1-col">
+  <header class="px-2 m:px-0">
     <h1>Top 100 Tags</h1>
-  </div>
-</div>
+  </header>
 
-<div class="home tag-home">
-  <div class="side-bar"></div>
-  <div class="articles-list" id="articles-list">
+  <div class="grid gap-2 m:gap-4 l:gap-6 m:grid-cols-2 l:grid-cols-3 px-2 m:px-0">
     <% @tags.each do |tag| %>
-      <div class="tag-list-container" style="background:<%= tag.bg_color_hex %>;">
-        <a class="tag-show-link" href="/t/<%= tag.name %>" style="color:<%= tag.text_color_hex || "#fff" %>;">
-          <h2>#<%= tag.name %></h2>
+      <% color = HexComparer.new([tag.bg_color_hex || "#0000000", tag.text_color_hex || "#ffffff"]).brightness(0.8) %>
+      <div class="tag-card crayons-card p-4 m:p-6  flex flex-col relative" style="border: 2px solid <%= color %>; box-shadow: 2px 2px 0 <%= color %>;">
+        <h3 class="crayons-tag crayons-tag--l mb-2">
+          <a href="/t/<%= tag.name %>" class="crayons-link">
+            <span class="crayons-tag__prefix">#</span><%= tag.name %>
+          </a>
+        </h3>
+        <% if tag.short_summary.present? %>
+          <p class="mb-2 truncate-at-3"><%= sanitize tag.short_summary %></p>
+        <% end %>
+        <p class="mb-3 fs-s color-base-50">
+          <%= pluralize @num_published_articles, "post" %> published
```

Comment:
I don't think this is going to work, it comes from a different controller.
You can use `tag.taggings_count`